merge.py

```diff
@@ -0,0 +1,188 @@
+"""Mergebot is a script which talks to GitHub and submits all ready pull requests.
+
+Mergebot talks to a specified GitHub project and watches for @mentions for its account.
+Acceptable commands are:
+  @<mergebot-name> merge
+"""
+from subprocess import call
+import glob
+import requests
+import sys
+import time
+
+AUTHORIZED_USERS = ["davor"]
+BOT_NAME = 'beam-testing'
+GITHUB_ORG = 'apache'
+REPOSITORY = 'incubator-beam'
+SECRET_FILE = '../../github_auth/apache-beam.secret'
+SOURCE_REMOTE = 'github'
+TARGET_BRANCH = 'master'
+TARGET_REMOTE = 'apache'
+
+GITHUB_API_ROOT = 'https://api.github.com'
+GITHUB_REPO_FMT_URL = GITHUB_API_ROOT + '/repos/{0}/{1}'
+GITHUB_REPO_URL = GITHUB_REPO_FMT_URL.format(GITHUB_ORG, REPOSITORY)
+CMDS = ['merge']
+ISSUES_URL = GITHUB_REPO_URL + '/issues'
+COMMENT_FMT_URL = ISSUES_URL + '/{pr_num}/comments'
+PULLS_URL = GITHUB_REPO_URL + '/pulls'
+
+bot_key = ''
+
+
+def main():
+  print('Starting up.')
+  # Load github key from filesystem
+  configs = {}
+  for filename in glob.iglob("config/*.json"):
```

Comment:
I think let's hold off for now since this will be a bigger issue than just throttling based on number of configs.